### ISSUE TITLE
Change the scope of `spring-boot.ls.java.home` to `machine-overridable`

### ISSUE THREAD

**Issue by cypher256** [spring-projects/sts4#1094]
Is it possible to change the scope of `spring-boot.ls.java.home` to `machine-overridable`?
I would like to have remote `settings.json` updated when `vscode.workspace.getconfiguration()` is `updated` by an extension in a remote environment such as WSL.

* Related Issue
https://github.com/redhat-developer/vscode-java/issues/2001
https://github.com/redhat-developer/vscode-java/pull/2368

**Comment by BoykoAlex** [comment]
Yep :-) Sorry just saw this issue... looks dumb-simple to me and should get this done for the next release.

**Comment by BoykoAlex** [comment]
If I understood correctly I think this should do it: bf675bca7a9606576587cf539930926114504e30
